### PR TITLE
[1.32.0] Alert history

### PR DESCRIPTION
**Pull request by idmyn**
[preview deployment](https://conduktor-docs-git-alert-history-conduktor.vercel.app/platform/navigation/settings/alerts/)

I think it might be worth adding a section for "alert behaviour" but I'm personally not confident enough in all the details to write it up. I'm trying to include details as I find them though (like notifications being sent every hour while an alert stays in a 'firing' state). Or perhaps we just need to link to cortex docs somewhere..?

### CODE REVIEW

**Comment by idmyn** [comment]
> > I think it might be worth adding a section for "alert behaviour" but I'm personally not confident enough in all the details to write it up. I'm trying to include details as I find them though (like notifications being sent every hour while an alert stays in a 'firing' state). Or perhaps we just need to link to cortex docs somewhere..?
> 
> It would be good to not link to other docs pages, if you can avoid it.

@RG-conduktor is this true even if the other docs pages are external? In this case I think there is some behaviour defined by a component we don't control, so if we add details to our docs then there's a risk it could become inaccurate without us realising

File: docs/platform/navigation/settings/alerts.md [hunk 1]
```diff
@@ -28,15 +28,15 @@ import AlertCreationButton from './assets/create-alert-button.png';
...
+Alerts need a unique name, a metric to measure, an operator & value for their threshold and an owner.
+Currently alerts can be owned by an application instance, a group, or an individual user, and this determines who can edit the alert.
 **If the owner of an alert is deleted, then the alerts associated with that owner will also be deleted.**
 
 import AlertCreation from './assets/create-alert.png';
 
 <img src={AlertCreation} alt="Alert creation" style={{ width: 800, display: 'block', margin: 'auto', marginBottom: '20px' }} />
 
-If you have configured alerting integrations in [the integration settings page](/platform/navigation/settings/integrations) you will be able to set an external destination for alert notifications. 
+If you have configured alerting integrations in [the integration settings page](/platform/navigation/settings/integrations) you will be able to set an external destination for alert notifications.
```
[the remark this answers is not in the document]
Comment (idmyn) [reply docs/platform/navigation/settings/alerts.md:40]: In my mind this link is more of a convenience than 'necessary reading'

I could add something here like `(e.g. Slack/Teams/Webhooks)` ?

File: docs/platform/navigation/settings/alerts.md [hunk 2]
```diff
@@ -28,15 +28,15 @@ import AlertCreationButton from './assets/create-alert-button.png';
...
 
 Alerts are periodically checked for **every minute**.
```
[the remark this answers is not in the document]
Comment (idmyn) [reply docs/platform/navigation/settings/alerts.md:41]: I guess you're suggesting to remove this because it's too much detail? I can imagine some users might ask this kind of thing but perhaps not many - is there a rough guideline to follow on how much info is too much info?

I do understand that if we documented every small detail then the page would be massive and you'd miss the actually important detail but that line of "too much" is a hard one to draw

File: docs/platform/navigation/settings/alerts.md [hunk 3]
```diff
@@ -1,86 +1,59 @@
...
 ## Overview
 
-Our alerting solution enables you to react to Kafka infrastructure and application changes.
-Currently, we integrate with **Slack** and **MS Teams** so that you can receive real-time notifications.
-You can also set up arbitrary webhook destinations for alerts with any URL and custom headers.
+Alerts allow you to be notified and react to Kafka infrastructure or application changes as soon as they happen.
 
-You can find information about configuring alerting integrations in [our docs for the integrations page](/platform/navigation/settings/integrations).
-If you require an additional integration, please [let us know](https://conduktor.io/roadmap) so we can consider it.
+You can integrate alerts with **Slack** and **MS Teams** to receive real-time notifications. Alternatively, set up arbitrary webhook destinations with any URL and custom headers. Find out how to [configure integrations](/platform/navigation/settings/integrations).
```
Comment (AurelieMarcuzzo) [inline docs/platform/navigation/settings/alerts.md:13]:
```suggestion
You can integrate alerts with **Slack** and **MS Teams** to receive real-time notifications. Alternatively, set up arbitrary **webhook** destinations with any URL and custom headers. Find out how to [configure integrations](/platform/navigation/settings/integrations).
```

File: docs/platform/navigation/settings/alerts.md [hunk 4]
```diff
@@ -1,86 +1,59 @@
...
 
-## Alert creation
+## Create alerts
+Alerts can be created for most resources (e.g. brokers or topics) in Conduktor Console. 
 
-Click the bell icon on a graph to open the alert creation drawer.
+To create an alert:
+1. Navigate to the required resource.
+1. Click on the **Alerts** tab.
+1. Click **Create an alert**:
```
Comment (AurelieMarcuzzo) [inline docs/platform/navigation/settings/alerts.md:23]:
```suggestion
1. Navigate to the required resource.
2. Click on the **Alerts** tab.
3. Click **Create an alert**:
```

File: docs/platform/navigation/settings/alerts.md [hunk 5]
```diff
@@ -1,86 +1,59 @@
...
 import AlertCreationButton from './assets/create-alert-button.png';
 
-<img src={AlertCreationButton} alt="Alert creation button" style={{ width: 800, display: 'block', margin: 'auto', marginBottom: '20px' }} />
+<img src={AlertCreationButton} alt="Alert creation" style={{ width: 800, display: 'block', margin: 'auto', marginBottom: '20px' }} />
 
-Alerts need a unique name, a metric to measure, an operator & value for their threshold and an owner.  
-Currently alerts can be owned by an application instance, a group, or an individual user, and this determines who can edit the alert.  
-**If the owner of an alert is deleted, then the alerts associated with that owner will also be deleted.**
+In the **New Alert** pane enter the required details. Each alert has to have:
+  - unique name
```
Comment (AurelieMarcuzzo) [inline docs/platform/navigation/settings/alerts.md:30]:
```suggestion
  - a unique name
```

File: docs/platform/navigation/settings/alerts.md [hunk 6]
```diff
@@ -1,86 +1,59 @@
...
+  - a metric you want to measure (e.g. messageCount)
+  - an operator and value for the threshold
+  - an owner. This can be a group, an application instance or an individual user. The ownership will determine who can edit the alert.
+:::warning
+If the owner of an alert is deleted, the associated alerts will also be deleted.
+:::
+- (optional) comments lets you add a description explaining what the alert does or why you created it. This is useful if you're creating an external alert.
```
Comment (AurelieMarcuzzo) [inline docs/platform/navigation/settings/alerts.md:37]:
```suggestion
- (optional) a description explaining what the alert does or why you created it. This is useful if you're creating an external alert.
```

File: docs/platform/navigation/settings/alerts.md [hunk 7]
```diff
@@ -1,86 +1,59 @@
...
+- a destination: internal (Conduktor Console) or external (Slack, Teams or Webhook). Each alert can have **only one external destination**. To verify that it works, click **Send test**.
 
-import AlertCreation from './assets/create-alert.png';
+## Manage alerts
+You can disable an alert temporarily without deleting it. You can also test the alert at any point or duplicate it (e.g. if you want the same alert to go to different external destinations). 
 
-<img src={AlertCreation} alt="Alert creation" style={{ width: 800, display: 'block', margin: 'auto', marginBottom: '20px' }} />
+Select the event and click the three docs in the top-right corner:
```
Comment (AurelieMarcuzzo) [inline docs/platform/navigation/settings/alerts.md:43]:
```suggestion
Select the event and click the three dots in the top-right corner:
```

File: docs/platform/navigation/settings/alerts.md [hunk 8]
```diff
@@ -1,86 +1,59 @@
...
 
-Alerts are periodically checked for **every minute**.
+<img src={EditAlert} alt="Editing alerts" style={{ width: 200, display: 'block', margin: 'auto', marginBottom: '20px' }} />
+
+## Alerting history
+Each alert will keep a history of when it was triggered and the status over time. This gives you an overview of successful or failed deliveries. Alerts with the `firing` status will trigger a notification every hour.
+
+import AlertDetails from './assets/alert-details.png';
+
+<img src={AlertDetails} style={{ width: 800, display: 'block', margin: 'auto', marginBottom: '20px' }}
```
Comment (AurelieMarcuzzo) [inline docs/platform/navigation/settings/alerts.md:54]: I'd suggest we crop this image to only focus on what's important, so I'd get rid of the left side pane. That way we have a better view on the alerts history.

**Review by AurelieMarcuzzo — COMMENTED** [review]
Very good job 👍 I added some suggestions

File: docs/platform/navigation/settings/integrations.md [hunk 9]
```diff
@@ -1,20 +1,22 @@
...
+## Configuring integrations
+
+Conduktor Console can integrate with Slack, MS Teams and any system that receives webhooks.
 
-Conduktor console can integrate with Slack, MS Teams, and any system that receives webhooks.
-MS Teams and webhook integrations can be enabled and disabled from this page with a simple toggle.  
-**Disabling an integration used by existing alerts will not delete the alerts, but will prevent them from sending notifications externally.**  
 Enabling the **Slack** integration requires creating a Slack application, installing this to your Slack workspace, inviting your app to the channel and adding the application token to Conduktor.  
+
 Enabling the **MS Teams** integration requires a webhook URL from Teams when creating alerts with this destination.
```
Comment (idmyn) [inline docs/platform/navigation/settings/integrations.md:13]: @RG-conduktor thoughts?

```suggestion
After enabling the **MS Teams** integration you will need a Teams webhook URL to create alerts with this destination.
```

File: docs/platform/navigation/settings/alerts.md (hunk 4 above)
Comment (AurelieMarcuzzo) [reply docs/platform/navigation/settings/alerts.md:23]: Oooh i didn't know! Very interesting, thanks :)
